src/components/media-player/media-player.component.spec.ts

```diff
@@ -0,0 +1,76 @@
+        component.displayName = '';
+        fixture.detectChanges();
+        expect(getTitle().innerText).toBeFalsy();
+    });
+
+    it('should display displayName when one is added to the markup', () => {
+        component.displayName = 'Display Name';
+        fixture.detectChanges();
+        expect(getTitle().innerText).toBeTruthy();
```

Comment:
Shouldn't this be something like `expect(getTitle().innerText).toBe('Display Name')`?